CHANGELOG.md

```diff
@@ -1,3 +1,52 @@
+## [0.13.2](https://github.com/act3-ace/safe-autonomy-dynamics/compare/v0.13.1...v0.13.2) (2023-09-21)
```

Comment:
Will these tags be valid?

Reply:
They will not be.  The scrubbing of the repo caused the change in the URLs to something that doesn't exist.  If they weren't changed, they would have pointed to something the public can't access.  We could either remove the links from the change log, with future releases adding valid links to new changes, or we could remove the log completely.  Then, we'd start with a fresh change log, but only containing information for the new changes.  I'd probably lean toward removing the links as I could see someone wanting to know what has changed since the last public release.

Reply:
Yes I agree we definitely should keep the changelog. I'm in favor of removing the links completely.

Reply:
It looks like those are valid urls as far as GitHub compare is concerned, the tags just don't exist. I don't think it's worth trying to keep the tags synced just to make the links work.

Reply:
Links were removed.